src/main/java/ch/njol/skript/expressions/ExprArmorChangeItem.java

```diff
@@ -0,0 +1,63 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.expressions.base.EventValueExpression;
+import ch.njol.skript.lang.EventRestrictedSyntax;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.registrations.EventValues;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import com.destroystokyo.paper.event.player.PlayerArmorChangeEvent;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Armor Change Item")
+@Description("Get the unequipped or equipped armor item from a 'armor change' event.")
+@Examples({
+	"on armor change:",
+	"\tbroadcast the old armor item"
+})
+@RequiredPlugins("Paper")
+@Events("Armor Change")
+@Since("INSERT VERSION")
+public class ExprArmorChangeItem extends EventValueExpression<ItemStack> implements EventRestrictedSyntax {
+
+	static {
+		if (Skript.classExists("com.destroystokyo.paper.event.player.PlayerArmorChangeEvent"))
+			register(ExprArmorChangeItem.class, ItemStack.class,
+				"[the] (old|unequipped) armo[u]r item",
+				"[the] (new|equipped) armo[u]r item");
```

Comment:
```suggestion
				"(old|unequipped) armo[u]r item",
				"(new|equipped) armo[u]r item");
```
i know the method accounts for duplicates, but no reason to deliberately cause duplicates to just remove them later